OAuthSwiftDemo/ViewController.swift

```diff
@@ -408,14 +408,15 @@ extension ViewController {
         oauthswift.authorizeWithCallbackURL( NSURL(string: "http://oauthswift.herokuapp.com/callback/linkedin2")!, scope: "r_fullprofile", state: state, success: {
             credential, response, parameters in
             self.showTokenAlert(serviceParameters["name"], credential: credential)
-            self.testLinkedin2(oauthswift)
+            self.testLinkedin2(credential.oauth_token, oauthswift: oauthswift)
```

Comment:
no need to pass token to test method, could be retrieved from oauthswift object
best to keep same signature for all test method if possible (for easy copy paste)